addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/EvohomeApiClient.java

```diff
@@ -17,4 +18,5 @@
 
     ControlSystem getControlSystem(int id);
 
-}
\ No newline at end of file
+    ZoneStatus getHeatingZone(int locationId, int zoneId);
```

Comment:
Hi @foxy82, where you planning on making models for the interface ? If not, and we're going for v2 models, I can merge this one in.